scripts/alertmailer.php

```diff
@@ -13,6 +13,20 @@ function mlog($message) {
 ini_set('memory_limit', -1);
 include_once INCLUDESPATH . 'easyparliament/member.php';
 
+$announcement_manager = new \MySociety\TheyWorkForYou\Model\AnnouncementManagement();
+$banner = $announcement_manager->get_random_valid_item('alerts');
+
+$text_banner = '';
+$html_banner = '';
+if ($banner) {
+    $text_banner = $banner->title . "\n\n" . $banner->content . "\n\n" . $banner->url . "\n\n";
+    $link_text = "Read more";
+    if (property_exists($banner, 'button_text')) {
+        $link_text = $banner->button_text;
+    }
+    $html_banner = '<strong>' . $banner->title . "</strong><p>" . $banner->content . '</p><p><a href="' . $banner->url . '">' . $link_text . '</a></p>';
```

Comment:
Could we only output the second p if link_text isn't blank? People could put their own links inside the content presumably, and then not need a "Read more"? e.g. the last time we did this manually, the added text was:
```
 <p style="font-size: 16px;">
The UK Parliament is back – read our blog post on
<a href="https://www.mysociety.org/2024/07/04/theyworkforyou-understanding-the-new-parliament/">resources to help you understand the new Parliament</a>,
and if you enjoy your alerts,
<a href="https://www.theyworkforyou.com/support-us/">donate to support our work</a>.
            </p>
```